e2e/testdata/thin_provisioning/pod-template.yaml

```diff
@@ -3,12 +3,12 @@ kind: Pod
 metadata:
   name: %s
   labels:
-    app.kubernetes.io/name: ubuntu
+    app.kubernetes.io/name: pause
 spec:
   containers:
     - name: ubuntu
-      image: quay.io/cybozu/ubuntu:20.04
-      command: ["/usr/local/bin/pause"]
+      image: ubuntu:20.04
+      command: ["/usr/bin/sleep", "infinity"]
```

Comment:
can we use pause?

Reply:
Cloning test cases use some commands in this container.

https://github.com/topolvm/topolvm/blob/main/e2e/clone_test.go